### DISCUSSION TITLE
How to use sklearn and time in my file?

### DISCUSSION THREAD

**Discussion by xyrrrrrrrr** [cyberbotics/wrestling#46] [General]
Yesterday when I test my new algorithm with sklearn and time added in dockerfile some unknown errors happened. If someone can answer my question， that will be very kind of you！

**Comment by Jean-Eudes-le-retour** [comment]
Can you provide more information on the problem you are having?

Can you provide the errors you are getting?
Are the errors happening on your computer locally, on GitHub's automatic evaluation script or both?
If it happened on GitHub's automatic evaluation script, could you give a link to the logs of the GitHub Actions?

**Reply by xyrrrrrrrr** [reply]
![image](https://user-images.githubusercontent.com/102366791/222396160-1a0988d9-8eed-4b45-8685-a2e3edb20494.png)

This is the error log file, I don't understand what happen exactly because **the error never comes  before I start to use sklearn and time**, and **after I remove them, the error never happens.**   I'm quite confused.

 I'd appreciate it a lot if you can answer it!

**Reply by xyrrrrrrrr** [reply]
It only happens on GitHub's automatic evaluation script

**Reply by xyrrrrrrrr** [reply]
https://github.com/cyberbotics/wrestling/actions/runs/4298714267/jobs/7493148312

**Reply by Jean-Eudes-le-retour** [reply]
When building the Docker image, pip can't find a package named "time" ([relevant error line](https://github.com/cyberbotics/wrestling/actions/runs/4298714267/jobs/7493148312#step:6:61)). I think that you don't need to install any additional packages for `time` as it is already present in python.

**Reply by Jean-Eudes-le-retour** [reply]
Can you try and remove the installation of "time" by pip inside your Dockerfile?

**Reply by xyrrrrrrrr** [reply]
I see. Thank you for your answer!
